includes/Core/Form/Integrations/PopupMaker.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace WeDevs\WeMail\Core\Form\Integrations;
+
+use WeDevs\WeMail\Traits\Singleton;
+
+class PopupMaker extends AbstractIntegration {
+
+    use Singleton;
+
+    /**
+     * Integration title
+     *
+     * @since 1.0.0
+     *
+     * @var string
+     */
+    public $title = 'Popup Maker';
+
+    /**
+     * Integration slug
+     *
+     * @since 1.0.0
+     *
+     * @var string
+     */
+    public $slug = 'popup_maker';
+
+    /**
+     * Execute right after making class instance
+     *
+     * @since 1.0.0
+     *
+     * @return void
+     */
+    public function boot() {
+        if ( class_exists( 'Popup_Maker' ) ) {
+            $this->is_active = true;
+        }
+    }
+
+    /**
+     * Get available forms
+     *
+     * @since 1.0.0
+     *
+     * @return array
+     */
+    public function forms() {
+        $forms = [];
+        $popups = pum_get_all_popups();
+
+        foreach ( $popups as $popup ) {
+            $fields = $this->get_fields( $popup->post_content );
+            if ( ! $fields ) {
+                continue;
+            }
+
+            $forms[] = [
+                'id'     => $popup->ID,
+                'title'  => $popup->post_title ? $popup->post_title : 'no title',
+                'fields' => $fields,
+            ];
+        }
+
+        return $forms;
+    }
+
+    private function get_fields( $content ) {
+        $data = explode( '[', rtrim( $content, ']' ) );
+        if ( ! $data || ! isset( $data[1] ) ) {
+            return false;
+        }
+        $data = $data[1];
+        $data = str_replace( '"', "'", $data );
+        $data = stripcslashes( $data );
+        $data = explode( ' ', $data );
+
+        if ( ! $data || $data[0] !== 'pum_sub_form' ) {
+            return false;
+        }
+
+        $fields = [];
+        foreach ( $data as $item ) {
+            $item = explode( '=', $item );
+            if ( ! isset( $item[0] ) ) {
```

Comment:
Need to use `continue` or `break` statement, rather than `return false` inside of loop.